src/components/GalleryLightBox/GalleryLightBox.tsx

```diff
@@ -197,23 +198,27 @@ export const GalleryLightBox = ({
                         )}
                       </dl>
                     )}
-                    <div className="cc-gallery-lightbox__download">
-                      <a
-                        className="cc-gallery-lightbox__download-link u-color-inherit"
-                        download
-                        href={slide.mediaSources.gallery_full_hi}
-                        rel="noopener noreferrer"
-                        target="_blank"
-                      >
-                        <span className="cc-gallery-lightbox__download-icon">
-                          <Icon name="download" />
+                    {slide.download === 'On' && (
```

Comment:
I think this would work much better as a boolean e.g. `hasDownload` or `canDownload`

Reply:
It is a boolean, just realised at the moment currently value is 'true' or 'false'. How would I do with canDownload?

Reply:
> It is a boolean, just realised at the moment currently value is 'true' or 'false'. How would I do with canDownload?

just double checking that's not a string `'true' | 'false'` ?

Reply:
I think it's a string

Reply:
any chance it could be a boolean instead ? 😬

Reply:
It is a boolean. I've just checked it.